services/workflow.go

```diff
@@ -34,6 +36,15 @@ func init() {
 	FUNC_MAP = map[string]interface{}{
 		"ld_ld_sync": ld_ld_sync,
 	}
+
+	Workflows = []Workflow{
```

Comment:
Why do we need to initialize the workflow here? In line 16 in `init` we also create the same object. Seems to me like we should delete it.